robottelo/ui/user.py

```diff
@@ -86,22 +90,18 @@ def create(self, username=None, email=None, password1=None, password2=None,
             if self.wait_until_element(
                     locators['users.password_confirmation']):
                 self.field_update('users.password_confirmation', password2)
-            if locale:
-                Select(
-                    self.find_element(locators['users.language'])
-                ).select_by_value(locale)
-            if edit:
-                self._configure_user(
-                    roles=roles,
-                    locations=locations,
-                    organizations=organizations,
-                    default_org=default_org,
-                    default_loc=default_loc,
-                    select=select
-                )
-            if admin:
-                self.click(tab_locators['users.tab_roles'])
-                self.click(locators['users.admin_role'])
+        if edit:
+            self._configure_user(
+                roles=roles,
+                locations=locations,
+                organizations=organizations,
+                default_org=default_org,
+                default_loc=default_loc,
+                select=select
+            )
+        if admin:
+            self.click(tab_locators['users.tab_roles'])
+            self.click(locators['users.admin_role'])
```

Comment:
Also I like to comment that this ideally needs to be 2 PRs.

* to fix base.click() to  make `scroll=True` and fix `select()` method
* to fix user.py to move locale, edit, admin as this is a pure user.py fix.